### ISSUE TITLE
packages not being uploaded?

### ISSUE THREAD

**Issue by daler** [bioconda/recipes#21]
Travis tests appear to be running fine, but packages aren't being uploaded. For example, trying to create a new environment using the last couple of recipes committed fails, e.g., 

```bash
conda create -n testing -c bioconda httplib2
#Fetching package metadata: ........
#Error: No packages found in current linux-64 channels matching: httplib2
#
#You can search for this package on anaconda.org with
#
#    anaconda search -t conda httplib2
```
And on https://anaconda.org/bioconda, looks like SGA was the last to be uploaded, followed by pysam.

Any ideas on what's happening here?

**Comment by johanneskoester** [comment]
Yes, I think I found the problem and fixed it in pull request #22.
Hopefully, stuff will start to show up when that is merged.

**Comment by daler** [comment]
Thanks, that did the trick!
